content/en/docs/setup/independent/high-availability.md

```diff
@@ -10,513 +10,217 @@ content_template: templates/task
-{{% tab name="systemd" %}}
-1. First, install etcd binaries:
-     ```bash
-     ETCD_VERSION="v3.1.12" curl -sSL https://github.com/coreos/etcd/releases/download/${ETCD_VERSION}/etcd-${ETCD_VERSION}-linux-amd64.tar.gz | tar -xzv --strip-components=1 -C /usr/local/bin/
-     ```
-
-    It is worth noting that etcd v3.1.12 is the preferred version for Kubernetes v1.10. For other versions of Kubernetes please consult [the changelog](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG.md).
-
-    Also, please realize that most distributions of Linux already have a version of etcd installed, so you will be replacing the system default.
-
-1. Next, generate the environment file that systemd will use:
-     ```bash
-     touch /etc/etcd.env
-     echo "PEER_NAME=${PEER_NAME}" >> /etc/etcd.env
-     echo "PRIVATE_IP=${PRIVATE_IP}" >> /etc/etcd.env
-     ```
-
-1. Now copy the systemd unit file:
-     ```none
-     cat >/etc/systemd/system/etcd.service <<EOF
-     [Unit]
-     Description=etcd
-     Documentation=https://github.com/coreos/etcd
-     Conflicts=etcd.service
-     Conflicts=etcd2.service
-
-     [Service]
-     EnvironmentFile=/etc/etcd.env
-     Type=notify
-     Restart=always
-     RestartSec=5s
-     LimitNOFILE=40000
-     TimeoutStartSec=0
-
-     ExecStart=/usr/local/bin/etcd --name <name> --data-dir /var/lib/etcd --listen-client-urls http://localhost:2379 --advertise-client-urls http://localhost:2379 --listen-peer-urls http://localhost:2380 --initial-advertise-peer-urls http://localhost:2380 --cert-file=/etc/kubernetes/pki/etcd/server.pem --key-file=/etc/kubernetes/pki/etcd/server-key.pem --client-cert-auth --trusted-ca-file=/etc/kubernetes/pki/etcd/ca.pem --peer-cert-file=/etc/kubernetes/pki/etcd/peer.pem --peer-key-file=/etc/kubernetes/pki/etcd/peer-key.pem --peer-client-cert-auth --peer-trusted-ca-file=/etc/kubernetes/pki/etcd/ca.pem --initial-cluster <etcd0>=https://<etcd0-ip-address>:2380,<etcd1>=https://<etcd1-ip-address>:2380,<etcd2>=https://<etcd2-ip-address>:2380 --initial-cluster-token my-etcd-token --initial-cluster-state new
-
-     [Install]
-     WantedBy=multi-user.target
-     EOF
-     ```
-
-     Make sure you replace `<etcd0-ip-address>`, `<etcd1-ip-address>` and `<etcd2-ip-address>` with the appropriate IPv4 addresses. Replace `<name>` with the name of this etcd member. Modify the values of `--listen-client-urls`, `--advertise-client-urls`, `--listen-peer-urls` and `--initial-advertise-peer-urls` if needed. Replace `<etcd0>`, `<etcd1>` and `<etcd2>` with real hostnames of each machine. These machines must be able to reach every other using DNS or make sure that records are added to `/etc/hosts`.
-
-1. Finally, launch etcd:
-     ```bash
-     systemctl daemon-reload
-     systemctl start etcd
-     ```
-
-1. Check that it launched successfully:
-     ```bash
-     systemctl status etcd
-     ```
+Now we can take that value and put it in `~/.ssh/authorized_keys` of any other host we want to connect to.
```

Comment:
I'd add a snippet folks can use to copy keys around.  

```
for host in m1.example.com \
    m2.example.com \
    m3.example.com; \
    do ssh-copy-id -i ~/.ssh/id_rsa.pub $host; \
    done
```

Reply:
Yes. This should be another numbered step, too:

1. Add the returned value to `~/.ssh/authorized_keys` on the other hosts to connect to:
<Tim's snippet>